plugins/runtime/threads/threads.service.ts

```diff
@@ -40,6 +39,37 @@ namespace Runtime {
     disableThreadContentionMonitoring(): angular.IPromise<void> {
       return this.jolokiaService.setAttribute('java.lang:type=Threading', 'ThreadContentionMonitoringEnabled', false);
     }
+
+    dumpThreads(): angular.IPromise<string> {
+      let dumpedThreads = "";
+      return this.jolokiaService.execute('java.lang:type=Threading', 'dumpAllThreads(boolean, boolean)', true, true)
+        .then(threads => {
+          threads.forEach(thread => {
+            thread.threadState = ThreadsService.STATE_LABELS[thread.threadState];
+            thread.waitedTime = thread.waitedTime > 0 ? Core.humanizeMilliseconds(thread.waitedTime) : '';
+            thread.blockedTime = thread.blockedTime > 0 ? Core.humanizeMilliseconds(thread.blockedTime) : '';
+          });
+          for (let i = 0; i < threads.length; i++) {
+            let threadName = Core.escapeHtml(threads[i].threadName);
+            let threadId = Core.escapeHtml(threads[i].threadId.toString());
+            let threadPriority = Core.escapeHtml(threads[i].priority.toString());
+            let isDaemonThread = threads[i].daemon == true ? Core.escapeHtml("daemon") : Core.escapeHtml("");
```

Comment:
this is meaningless. you dont need to use escapeHtml here.

Reply:
why so?